### ISSUE TITLE
Create new trigger to deploy drupal content to content bucket

### ISSUE THREAD

**Issue by billfienberg** [department-of-veterans-affairs/va.gov-team#12324]
Before we can ensure the contents of the new content bucket are included in the original bucket, we need to add a trigger to deploy drupal content to the content bucket.

**Comment by billfienberg** [comment]
Closing: Duplicate of https://github.com/department-of-veterans-affairs/va.gov-team/issues/7971
